### ISSUE TITLE
I got an error when I train CenterNet2_R2-101-DCN-BiFPN_4x+4x_1560_ST 

### ISSUE THREAD

**Issue by marcovwu** [xingyizhou/CenterNet2#26]
I trained :
python ./tools/train_net.py --config-file ./configs/COCO-Detection/faster_rcnn_R_50_C4_1x.yaml --num-gpus 1 SOLVER.IMS_PER_BATCH 2 SOLVER.BASE_LR 0.0025
![image](https://user-images.githubusercontent.com/50776722/112680175-02597880-8ea8-11eb-9c98-6e2b49ac50be.png)
Susccessful!!
But I run : 
python ./projects/CenterNet2/train_net.py --config-file ./projects/CenterNet2/configs/CenterNet2_R2-101-DCN-BiFPN_4x+4x_1560_ST.yaml --num-gpus 1 SOLVER.IMS_PER_BATCH 2

![image](https://user-images.githubusercontent.com/50776722/112679051-b0642300-8ea6-11eb-9b25-b56a91866a1f.png)
There is an error, I don't know why?

**Comment by xingyizhou** [comment]
Hi,
  Sorry for the delayed reply. As you have noted, you will need to decrease the learning rate when reducing the batch size. Do you also encounter the same error when doing so for the CenterNet2 model?

**Comment by marcovwu** [comment]
Thanks!! I tried to load pretrained model (  CenterNet2_R2-101-DCN-BiFPN_4x+4x_1560_ST.pth ), which sloved my problem.

**Comment by sunset326** [comment]
Hi,i want to use CenterNet2_R2-101-DCN-BiFPN_4x+4x_1560_ST  model to demo ,but i can't find the file of Base-CenterNet2.yaml.how to download the file.
